ingest/expression_files/expression_files.py

```diff
@@ -138,6 +170,49 @@ def check_unique_cells(cell_names: List, study_id, client):
             raise ValueError(error_string)
         return True
 
+    @staticmethod
+    def has_expression_file_info_doc(study_id, study_file_id, client):
+        COLLECTION_NAME = "study_files"
+        QUERY = {
+            "study_id": study_id,
+            "study_file_id": study_file_id,
+            "expression_file_info": {"$exists": True},
+        }
+        query_results = list(client[COLLECTION_NAME].find(QUERY))
+        if query_results:
+            return True
+        return False
+
+    @staticmethod
+    def get_study_expression_file_ids(
```

Comment:
I named it this way because of this [comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/158#discussion_r511051170). If you'd still like it changed I'll be happy to switch it up!